spec/unit/recipes/_install_spec.rb

```diff
@@ -120,12 +119,11 @@
 
     context 'when platform is rhel' do
       let(:platform) { 'centos' }
-      let(:platform_version) { '6.6' }
+      let(:platform_version) { '6.9' }
 
       it 'installs downloaded splunk package and notifies splunk-first-run' do
         expected_attrs = {
-          source: splunk_filepath,
-          provider: Chef::Provider::Package::Rpm
```

Comment:
Are we getting rid of provider object specifications? I don't see a change to do that, but we're getting rid of these attributes...?

Reply:
Still going to be removed, Providers usage changed in [FC109](http://www.foodcritic.io/#FC109)

I will have changes reflecting this in my next commit.